__tests__/endpoints/resourcesGet.test.js

```diff
@@ -1,30 +0,0 @@
-import connect from 'mongo.js'
```

Comment:
Why did this test get deleted?

Reply:
It didn't. It's a rename - to resourceGet.test.js - the path is `resource` not `resources` so I renamed as it made more sense to me. Not sure why github is reporting this one as a delete but the other 2 as a rename.